joomla/joomla-cms
# [com_categories] Complete tooltip

#### Summary of Changes

Simple PR to improves the search tooltip and element label (accessibility) in com_categories.
#### Testing Instructions

Code review, or:
1. Apply patch
2. Go to Content -> Categories, check the tooltip in the search field.
